cache/image_cache.go

```diff
@@ -78,7 +73,7 @@ func (c *ImageCache) Commit() error {
 	}
 
 	c.origImage = c.newImage
-	c.newImage = c.factory.NewEmptyLocal(c.origImage.Name())
+	c.newImage = c.initializer(c.origImage.Name())
```

Comment:
I think we should keep the original origImage and cache image after saving

Reply:
Volume cache would need to change to behave analogously if we made the change I am suggesting